src/components/App.tsx

```diff
@@ -25,6 +25,9 @@ import { makeConsentManagerAPI } from '../api';
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const IntlProvider = _IntlProvider as any;
 
+// Create `transcend` eventTarget on the global scope so this isn't derefenced on the next render of App
+const eventTarget = new EventTarget();
```

Comment:
Notes for EdgeHTML & Safari <14 support: https://github.com/transcend-io/main/blob/202022566a4c85e28ad441eb78678fbb02327dab/consent-manager/airgap.js/src/core/event-target.ts#L19-L37